src/components/CarouselRulesForm/CarouselRulesForm.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react';
+import { TextInput } from '../TextInput';
+import { CarouselRules } from '../../types/CarouselRules';
+
+type Props = {
```

Comment:
```suggestion
interface  Props = {
```
If you can write interface instead of type, use it, this is the best way.